Makefile

```diff
@@ -62,7 +62,7 @@ test:  ## run the tests (except sdk tests)
 .PHONY: test
 
 test-acceptance: ## runs all tests, including the acceptance tests which create and destroys real resources
-	SKIP_MANAGED_ACCOUNT_TEST=1 TF_ACC=1 $(go_test) -v -coverprofile=coverage.txt -covermode=atomic $(TESTARGS) ./...
+	SKIP_MANAGED_ACCOUNT_TEST=1 SKIP_EMAIL_INTEGRATION_TESTS=1 TF_ACC=1 $(go_test) -timeout 900s -v -coverprofile=coverage.txt -covermode=atomic $(TESTARGS) ./...
```

Comment:
only 5.74s when i benchmark on my computer. i don't think that resource was the one causing the problem. more like we were on the cusp of going over 10 minutes for a while, and recently we have exceeded that since we have been adding more test cases. this one just happened to be the test that was running while the deadline was exceeded

Reply:
Oh that would make sense but then how do you explain the tests continuing after one failed? If it was really the test runner as a whole timing out I wouldn't expect it to continue executing tests.
![image](https://user-images.githubusercontent.com/128428373/236913096-5682132f-898e-4f8a-b467-706b550b7439.png)

Reply:
looks like the default 600 second limit applies per package.